app/controllers/SignOutController.scala

```diff
@@ -41,7 +41,7 @@ class SignOutController @Inject()(config: FrontendAppConfig,
     }
       .map(contactFormIdentifier => Redirect(config.ggSignOutUrl(contactFormIdentifier)))
       .recover {
-        case _: AuthorisationException => Redirect(config.signInUrl)
+        case _: AuthorisationException => Redirect(config.ggSignInUrl)
```

Comment:
We need to keep signInUrl val in frontend config which is deleted and point to that rather than ggsigninurl as it doesn't have redirect params in it.

lazy val signInUrl: String = servicesConfig.getString("base.sign-in")